src/Microsoft.PowerShell.ConsoleGuiTools/Microsoft.PowerShell.ConsoleGuiTools.csproj

```diff
@@ -6,7 +6,7 @@
 
     <ItemGroup>
         <PackageReference Include="Nstack.Core" Version="1.0.7" />
-        <PackageReference Include="Terminal.Gui" Version="1.10" />
+        <PackageReference Include="Terminal.Gui" Version="1.13" />
```

Comment:
@tig you also need to add the  `'publish/Microsoft.DotNet.PlatformAbstractions.dll'` in the `ModuleLayout.psd1` file, otherwise it'll fail.

```psd1
            'publish/NStack.dll',
            'publish/Microsoft.DotNet.PlatformAbstractions.dll'

```

If you prefer not using more dependencies I found a way to change the `Terminal.Gui` to found the OS name by only add:

```diff
+using Terminal.Gui;
...
-						if (Microsoft.DotNet.PlatformAbstractions.RuntimeEnvironment.OperatingSystem.Contains ("opensuse")) {
+						var (exitCode, result) = ClipboardProcessRunner.Bash ("cat /etc/os-release", waitForOutput: true);
+						if (exitCode == 0 && result.Contains ("opensuse")) {
							libs [0] = "libncursesw.so.5";
						}
```

Reply:
We should use this solution in Terminal.Gui and not force another dependency onto ConsoleGuiTools.

Reply:
Can you do a PR for this in T.Gui, @BDisp?

I'll then do a v1.13.2 release.

Reply:
Done in https://github.com/gui-cs/Terminal.Gui/pull/2753.